tests/openqa/osautoinst/test_running.pm

```diff
@@ -12,7 +12,8 @@ use base "consoletest";
 use testapi;
 
 sub run {
-    assert_script_run ' ret=false; for i in {1..5} ; do openqa-client jobs state=running | grep --color -z running && ret=true && break ; sleep 30 ; done ; [ "$ret" = "true" ] ; echo $? ', 300;
+    assert_script_run 'command -v ack >/dev/null || zypper --no-refresh -n in ack';
```

Comment:
Maybe it could be a good idea to move the installation somewhere else, where we can do retries / error-catching as zypper call tends to make the code less stable.

Reply:
Not sure about moving it. There's calls all over the test distri anyway - but we can use `zypper_call` of course which will retry up to 5 times among other things.